core/src/main/java/psiprobe/tools/AccessorFactory.java

```diff
@@ -43,7 +43,13 @@ public static Accessor getInstance() {
   private static Accessor getReflective() {
     try {
       return new ReflectiveAccessor();
-    } catch (Exception ex) {
+    } catch (ClassNotFoundException e) {
```

Comment:
Yeah, I noted elsewhere we'd have to bump the property values to 1.7 in the root pom.xml to use multi-catch. I don't see any problem with pulling the trigger on that now.